_includes/body-landing.html

```diff
@@ -0,0 +1,844 @@
+<body ng-app="Docker" ng-controller="DockerController" id="landing">
+  <nav class="navbar header">
+    <div class="container">
+      <div class="navbar-header">
+        <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#header-content" aria-expanded="false">
+          <span class="sr-only">Toggle navigation</span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+        </button>
+        <a class="navbar-brand" href="/">
+          <img
+            src="../images/docker-docs-logo.svg"
+            alt="Docker Docs"
+            title="Docker Docs"
+          />
+        </a>
+      </div>
+      <div class="collapse navbar-collapse" id="header-content">
+        <ul class="nav navbar-nav">
+          <li class="active">
+            <a href="https://docs.docker.com/">Home</a>
+          </li>
+          <li>
+            <a href="https://docs.docker.com/develop/">Guides</a>
+          </li>
+          <li>
+            <a href="https://docs.docker.com/engine/">Product Manual</a>
+          </li>
+          <li>
+            <a href="https://docs.docker.com/reference/">Reference</a>
+          </li>
+          <li>
+            <a href="https://www.docker.com/blog/">Articles</a>
+          </li>
+        </ul>
+        <ul class="nav navbar-nav navbar-right">
+          <li>
+            {% include theme-switch.html %}
+          </li>
+        </ul>
+      </div>
+    </div>
+  </nav>
+
+  <section class="hero container">
+    <div class="row justify-content-center">
+      <div class="col-xs-12 text-center">
+        <h2>
+          What can we help you find?
+        </h2>
+      </div>
+    </div>
+    <div class="row justify-content-center">
+      <form
+        class="col-xs-12 col-sm-offset-2 col-sm-8 col-md-offset-3 col-md-6 col-lg-offset-4 col-lg-4"
+        method="GET"
+        action="/search/"
+      >
+        <input
+          class="form-control"
+          name="q"
+          placeholder="Search for guides, terms, commands and more..."
+          type="search"
+          autocomplete="off"
+          spellcheck="false"
+          dir="auto"
+        />
+      </form>
+    </div>
+  </section>
+
+  <section class="container">
+    <div class="row">
+      <div class="col-xs-12 col-sm-6 col-lg-4 card-holder">
```

Comment:
After this is merged, we could have a look at separate includes for these tiles, then it'd be easier to "swap out" things in case we want to hightlight something else.

Reply:
lgtm